examples/graphql-subscriptions/graphql_subscriptions.bal

```diff
@@ -1,22 +1,32 @@
 import ballerina/graphql;
+import ballerina/random;
+import ballerina/lang.runtime;
 
 service /graphql on new graphql:Listener(9090) {
-    // Defines a `string` array in the service.
-    private string[] names;
 
-    function init() {
-        // Initialize the array.
-        self.names = ["Walter White", "Jesse Pinkman", "Skyler White"];
-    }
-
-    resource function get names() returns string[] {
-        return self.names;
+    resource function get greeting() returns string {
```

Comment:
What if we have names here and pass it as an argument to the stream implementor? That way we can give it both ways (and may make more sense than greeting?).

Reply:
+1